notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py

```diff
@@ -226,7 +226,7 @@
 
 # EPY: START code
 from starfish.constants import Indices
-from starfish.pipeline.segmentation.watershed import _WatershedSegmenter
+from starfish.image._segmentation import _WatershedSegmenter
```

Comment:
This is awkward as well.

I have this vague notion that we discussed this before (although maybe it was with @dganguli), and the right strategy is:

```
from starfish.image import Segmentation

segmenter = Segmentation.WatershedSegmenter(...)
segmenter.run(...)
```